compiler/rustc_llvm/Cargo.toml

```diff
@@ -14,3 +14,7 @@ libc = "0.2.73"
 # pinned `cc` in `rustc_codegen_ssa` if you update `cc` here.
 cc = "=1.2.16"
 # tidy-alphabetical-end
+
+[features]
+# Used by ./x.py check --compile-time-deps to skip building C++ code
```

Comment:
Why is this only set with compile-time-deps? A regular check build also doesn't need the LLVM libraries, does it?

Reply:
To not change existing behavior for `./x.py check`. If this is also set for `./x.py check`, that will no longer check if all C++ source files compile.

Reply:
With https://github.com/rust-lang/rust/pull/146596 you may be able to disable the LLVM backend entirely for stage 1 builds if `./x.py miri` were to use the bootstrap compiler for building the build scripts of the standard library.

Reply:
Oh, funky. But most of the time I do need codegen as well (even Miri does, for build scripts and proc macros -- not sure if those can just use a different stage).

Reply:
When bootstrap builds the standard library it uses different stages for build scripts and for the standard library itself. This does use a wrapper around rustc though.